.changes/unreleased/Fixed-20220315-105032.yaml

```
@@ -0,0 +1,5 @@
+kind: Fixed
+body: 'fix(cli): changie next enforces arg to be patch|minor|major'
```

No fix cli necessary I would just say
```suggestion
body: 'next command enforces argument to be major, minor or patch'
```